app/helpers/course/discussion/topics_helper.rb

```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 module Course::Discussion::TopicsHelper
   include Course::Discussion::CodeDisplayHelper
+
+  # Sanitize the title in settings.
+  #
+  # @return [String|nil] The formatted title.
+  def topics_title
+    # We don't want to return a blank string so this check is necessary.
+    @settings.title ? format_inline_text(@settings.title) : nil
```

Comment:
Nope, here I mean `format_inline_text(nil)` will give `""`. It's same for every component: https://github.com/Coursemology/coursemology2/blob/master/app/helpers/course/announcements_helper.rb#L8